### ISSUE TITLE
yolov9 for custom model on macos just sucks?!!!!

### ISSUE THREAD

**Issue by R4Ajeti** [WongKinYiu/yolov9#553]
Training a model based on my experience it's just so terrible, 
1. Quantization does not work 
2. Running inference using onnxruntime does not work
3. Converting .onnx model to .tflite does not work

DO NOT USE THIS PACKAGE FOR CUSTOM MODEL
PS: I move to yolov10 to try it from tomorrow

**Comment by WongKinYiu** [comment]
https://github.com/WongKinYiu/yolov9?tab=readme-ov-file#useful-links

**Comment by kelderewrtiokh** [comment]
"The clear, step-by-step instructions on how to make [rosemary water](https://rosemarywater.site/) are incredibly helpful. I appreciate how easy your site is to navigate."

**Comment by meitueappfullversion** [comment]
If you're seeking to [explore APK archives](https://meeitueapk.com/) for the Meitu app, you're in luck. MeitueAPK.com offers a comprehensive collection of Meitu's previous versions, allowing users to download and install the iteration that best suits their device compatibility or personal preferences. This is particularly beneficial for users who may experience issues with the latest updates or prefer the functionality of earlier versions. By accessing these archives, you ensure a seamless photo editing experience tailored to your needs.

**Comment by lilymorgan430-rgb** [comment]
Yeah, YOLOv9 setup on macOS is really tough — lots of compatibility issues. Hope the next release improves it. For useful study and tech resources, visit [education in france](https://educonnectnormandie.online)

**Comment by lilymorgan430-rgb** [comment]
I’ve noticed similar issues while trying to train custom models with YOLOv9 on macOS — compatibility and dependency errors can be frustrating. The best solution is often setting up a clean virtual environment or switching to a Linux-based system for smoother GPU support. If you’re exploring AI tools, ML workflows, or performance optimization tips, check out [Rankdix](https://rankdix.com/)
 — they share helpful guides on technology, AI, and SEO-related topics.
